src/axom/core/tests/core_bit_utilities.hpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "gtest/gtest.h"
```

Comment:
We didn't have any tests for the bit utility functions: `popCount()`, `trailingZeros()` and `leadingZeros()`, so I added new tests before making changes.